pkg/store/store.go

```diff
@@ -122,6 +122,7 @@ type CertificateStore interface {
 type AnnouncementStore interface {
 	RetrieveAnnouncementMonth(date string) (*bff.AnnouncementMonth, error)
 	UpdateAnnouncementMonth(m *bff.AnnouncementMonth) error
+	DeleteAnnouncementMonth(date string) error
```

Comment:
We are not deleting announcement months in production code currently but this makes the tests a bit easier.